src/MongoDB.Analyzer/Core/Builders/BuilderExpressionProcessor.cs

```diff
@@ -106,16 +103,25 @@ public static ExpressionsAnalysis ProcessSemanticModel(MongoAnalyzerContext cont
             }
         }
 
-        var linqAnalysis = new ExpressionsAnalysis()
+        try
```

Comment:
Kill switch.

Reply:
Done. Kill Switch implemented in MongoDBAnalyzerSettings